### DISCUSSION TITLE
Hichi WiFi -> nosense values

### DISCUSSION THREAD

**Discussion by na-am** [arendst/Tasmota#20058] [Q&A]
hi@all
from time to time i get nosense values for a couple of minutes then it returns to 'normal' values:
![image](https://github.com/arendst/Tasmota/assets/47410115/48f422c3-086a-4ee6-97d5-0bcc641c2a17)

is this smart meter releated or is the Hichi a bit misplaced on it?

thx for hints
na-am

**Comment by Jason2866** [comment]
Since crystal balls are hard to source we can't help without any usefull infos

**Comment by thc08** [comment]
My crystal ball just came back from repairs. ;-)
I had exactly the same problems with a DH302.
It helps to cover the receiving diode with foil. Two layers of window film helped here.
To darken and reduce UV radiation. The head is too sensitive or the diode in the meter is too bright.
Maybe it will help someone else.
